### ISSUE TITLE
Double setting window opened

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#12994]
### Steps to reproduce:
* Press NVDA+control+G to open the General settings
* Press control+Tab to switch to Speech settings panel
* Press NVDA+control+G

### Actual behavior:
A second MultiCategorySettingsDialog is opened and the following error appears in the log.

```
IO - inputCore.executeGesture (00:17:51.115) - winInputHook (232):
Input: kb(desktop):NVDA+control+g
DEBUG - gui.settingsDialogs.__new__ (00:17:51.134) - MainThread (3304):
Creating new settings dialog (multiInstanceAllowed:False). State of _instances {<gui.settingsDialogs.NVDASettingsDialog object at 0x08453D50>: <DialogState.DESTROYED: 1>}
ERROR - gui.settingsDialogs.__new__ (00:17:51.135) - MainThread (3304):
Opening new settings dialog while instance still exists: <gui.settingsDialogs.NVDASettingsDialog object at 0x08453D50>

```

### Expected behavior:

Only one multi-category settings dialog at a time.

#### NVDA installed/portable/running from source:
installed
#### NVDA version:
2021.2
#### Windows version:
Windows 10 2004 (x64) build 19041.388
#### Does the issue still occur after restarting your computer?
Not yet exactly tested but probably yes.
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Last alpha from source (3ce266b2eb7efdab52b9d7fdff7e3383b2281a5a) gives the same issue.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested, probably unrelated.

**Comment by seanbudd** [comment]
I just tested this on alpha, and it appears to be fixed. Can you confirm?

Closed by #13117

**Comment by CyrilleB79** [comment]
Yes, I confirm that the error does not occur anymore.
At the third step, nothing happens, i.e. the speech settings of the current window remain opened.
